src/routes/+page.server.js

```diff
@@ -12,4 +13,34 @@ export const actions = {
     cookies.delete('sid', { path: '/', httpOnly: true, sameSite: 'lax' });
     redirect(307, '/');
   },
+  async dummy({ locals: { db, session }, cookies }) {
+    if (typeof session === 'undefined') error(401);
+    if (!dev) error(403);
+
+    const dummyId = crypto.randomUUID();
+    const emailLeader = dummyId.slice(0, 8);
+    const dummyEmail = `${emailLeader}@dummy.com`;
+
+    // log the current user out
+    const deleted = await db.deleteValidSession(session.id);
+    if (typeof deleted === 'undefined')
+      db.logger.warn('attempt to delete non-existent/expired session');
+    else db.logger.info({ deleteValidSession: deleted });
+
+    cookies.delete('sid', { path: '/', httpOnly: true, sameSite: 'lax' });
+
+    // log the dummy user in
+    db.logger.warn({ dummyEmail }, 'inserting dummy user')
+    const dummyUserId = await db.upsertOpenIdUser(
+      dummyEmail,
+      crypto.randomUUID(),
+      "Dummy",
+      `${emailLeader}`,
```

Comment:
No need for string interpolation here.
```suggestion
      emailLeader,
```